whelktool/src/main/groovy/datatool/WhelkTool.groovy

```diff
@@ -134,6 +134,7 @@ class WhelkTool {
         def segment = '/scripts/'
         def path = scriptFile.toURI().toString()
         path = path.substring(path.lastIndexOf(segment) + segment.size())
+        // FIXME: de-KBV/Libris-ify
```

Comment:
Nice to have

Reply:
Yes, we'll chalk this up to the quite improvable provenance handling.